inc/Lazyload/Component.php

```diff
@@ -144,6 +154,44 @@ public function action_enqueue_lazyload_assets() {
 		wp_script_add_data( 'wp-rig-lazy-load-images', 'precache', true );
 	}
 
+	/**
+	 * Prints the lazy-loading script.
+	 *
+	 * If the browser supports native lazy-loading via the 'loading' attribute, values of data attributes on the images
+	 * will be written back into their real attributes. If the browser does not support native lazy-loading, the script
+	 * bundled in the theme will be loaded.
+	 *
+	 * @see https://web.dev/native-lazy-loading
+	 */
+	public function action_print_lazyload_script() {
+		?>
+		<script type="text/javascript">
+			if ( 'loading' in HTMLImageElement.prototype ) {
+				document.addEventListener( 'DOMContentLoaded', function() {
```

Comment:
What about images loaded after the page has loaded, such as via an infinite scroll feature?

Reply:
I think that is something to revisit separately, as it's not covered by the current way the JS script works either.